docs/source/top_level_elements/originInfo_publishing.rst

```diff
@@ -0,0 +1,101 @@
+originInfo: publishing information
+==================================
+
+About
+-----
+
+This section describes the non-date specific subelements of originInfo in our MODS.
+
+publisher
+---------
+
+An example, `utsmc:13759 <https://digital.lib.utk.edu/collections/islandora/object/utsmc%3A13759>`_:
+
+.. code-block:: xml
+
+    <originInfo>
+        <place>
+            <placeTerm valueURI="http://id.loc.gov/authorities/names/n79006530">Baltimore (Md.)</placeTerm>
+        </place>
+        <publisher>Frederick D. Benteen</publisher>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators>
+
+    <https://example.org/objects/1> relators:pbl "Frederick D. Benteen" ;
+        relators:pup <http://id.loc.gov/authorities/names/n79006530> .
+
+.. code-block:: xml
+
+    <originInfo>
+      <publisher>Archivision, Inc.</publisher>
+      <dateCreated>begun 1665</dateCreated>
+      <dateCreated encoding="edtf" keyDate="yes">1665</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators>
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> relators:pbl "Archivision, Inc." ;
+        dcterms:created "1665/" .
```

Comment:
Not sure how we're going to be able to handle this programmatically (translating "begun" to the start of a timespan), but I agree with the modeling of this information.

Reply:
You're absolutely right: we're not! I updated the example to reflect our latest decisions on how to proceed.